rpc/amqp-rpc.go

```diff
@@ -241,12 +249,13 @@ type RPCResponse struct {
 
 // Start starts the AMQP-RPC server running in a separate thread.
 // There is currently no Stop() method.
-func (rpc *AmqpRPCServer) Start() (err error) {
+func (rpc *AmqpRPCServer) Start() (finished chan bool, err error) {
```

Comment:
Document what the finished chan means. Also, does the HandleInterrupts mechanism obviate the comment saying there is no Stop()?